src/dotnet/ToolPackage/CommandSettingsListId.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using NuGet.Frameworks;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.ToolPackage
+{
+    /// <summary>
+    ///     Given the following parameter, a list of CommandSettings of a NuGet package can be uniquely identified
+    /// </summary>
+    internal class CommandSettingsListId : IEquatable<CommandSettingsListId>
```

Comment:
Add CommandName property, rename to `CommandSettingsIdentifier`, or also rename `CommandSettings`, and rename this to something like `InstalledCommandIdentifier`.

Reply:
Renamed as suggested